settings.js

```diff
@@ -120,6 +156,16 @@ function generate(){
           "mappings" : punctuation.blacklist.map(function(c){
             return c + '=>';
           })
+        },
+        "alphanumeric" : {
+          "type" : "pattern_replace",
+          "pattern": "[^a-zA-Z0-9]",
+          "replacement": ""
+        },
+        "numeric" : {
+          "type" : "pattern_replace",
+          "pattern": "[^0-9]",
+          "replacement": " "
```

Comment:
Why do we remove non alphanumeric characters (by replacing them with empty string), but replace non numeric characters with a space? (I'm probably missing something, not actually saying we should change how it's done)